test/test_oauth2.py

```diff
@@ -80,44 +80,44 @@ def test_it_redirects_with_the_original_state_when_authorizing(test_client: Flas
                          'client_id': 'client_id', 'original': 'foo'})})
 
 
-def test_it_requires_a_code_when_checking(test_client: FlaskClient):
+def test_it_requires_a_code_when_checking(test_client: FlaskClient) -> None:
     response = test_client.get('/oauth2/check')
 
     assert response.status_code == 400
     assert 'Invalid code' in response.data.decode('UTF-8')
 
 
-def test_it_requires_a_state_when_checking(test_client: FlaskClient):
+def test_it_requires_a_state_when_checking(test_client: FlaskClient) -> None:
```

Comment:
is that needed for every method? :java:

Reply:
(considering no code calls test methods explicitly)

Reply:
this is the same kind of thinking that leads to OO and getters and setters and code generation and mutable state hiding in deeply nested class hierarchies and .... argh! I have to ignore this code, it's making me angry.

Reply:
OO has been the opposite of getters and setters for the last 14 years https://www.javaworld.com/article/2073723/core-java/why-getter-and-setter-methods-are-evil.html :)